business/products.md

```diff
@@ -0,0 +1,47 @@
+---
+description: Providers of machine translation services
+---
+
+# Providers
+
+Machine translation **providers** offer ready-to-use machine translation systems. Typically the systems are provided as APIs, and also as applications or integrations.
+
+They vary by:
+
+- language support
+- accuracy
+- price
+- customization options
+- data confidentiality
+
+and other features, as well as underlying architecture.
+
+
+The advantage of using a machine translation provider is that the service is already working and can be integrated immediately. The alternative is to train a model from scratch or to use a pre-trained model, and deploy and maintain a machine translation system.
+
+Unlike raw models, production machine translation systems usually provide language identification and sentence segmentation.
+
+Providers offer machine translation services via a platform, an API or both. Some providers offer limited services for free or paid full services.
+
+| Company | Product | Service |
```

Comment:
I would try to make this more structured.

So make *Self-serve* and *API* and *Custom* columns, and then it's just a binary value.

But I would remove "Custom" and "API", because almost all of them offer that.

So the only thing that needs to be there is "Self-serve".  Therefore, it would also be possible to split them into 2 lists.